pkg/tbtc/node.go

```diff
@@ -1075,6 +1089,144 @@ func processCoordinationResult(node *node, result *coordinationResult) {
 	}
 }
 
+// archiveClosedWallets archives closed or terminated wallets.
+func (n *node) archiveClosedWallets() error {
```

Comment:
I think we can simplify and optimize the `archiveClosedWallets` function by removing the `getClosedWallets` inner function and using a single `for` loop. What do you think?

Reply:
Done in 88f3bf6b3041e6f24f8b1cbcee6d17de4d6151ec.